### PR TITLE
Update remove-former-employee-step-4.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details on how to grant access to another user on former employees onedrive

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5087

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 1]
```diff
@@ -48,6 +48,31 @@ To preserve a former user's OneDrive files, first give yourself access to their
 > You can move or copy up to 500 MB of files and folders at a time.<br/>
 > When you move or copy documents that have version history, only the latest version is moved.  
 
+You can also grant access to another user on former employee's onedrive
+
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
You can also grant access to another user to a former employee's OneDrive.
```

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 2]
```diff
@@ -48,6 +48,31 @@ To preserve a former user's OneDrive files, first give yourself access to their
...
+3. If the classic SharePoint admin center appears, select **Open it now** at the top of the page to open the SharePoint admin center.
+
+4. In the left pane, select **More features**.
+
+5. Under **User profiles**, select **Open**.
+
+6. Under **People**, select **Manage User Profiles**.
+
+7. Enter the Former employee's name and select **Find**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
7. Enter the former employee's name and select **Find**.
```

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 3]
```diff
@@ -48,6 +48,31 @@ To preserve a former user's OneDrive files, first give yourself access to their
...
+
+8. Right-click the user, and then choose **Manage site collection owners**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md:70]:
```suggestion
8. Right-click on the user, and then choose **Manage site collection owners**.
```

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 4]
```diff
@@ -48,6 +48,31 @@ To preserve a former user's OneDrive files, first give yourself access to their
...
+
+9. Add the **User** to the **Site collection Administrators** and Click On **Ok**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
9. Add the user to **Site collection administrators** and select **Ok**.
```

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 5]
```diff
@@ -48,6 +48,31 @@ To preserve a former user's OneDrive files, first give yourself access to their
...
+
+10. The **User** will now able to access the former employees Onedrive using the onedrive **URL**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
10. The user will now be able to access the former employee's OneDrive using the OneDrive URL. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 6]
```diff
@@ -48,6 +48,30 @@ To preserve a former user's OneDrive files, first give yourself access to their
 > You can move or copy up to 500 MB of files and folders at a time.<br/>
 > When you move or copy documents that have version history, only the latest version is moved.  
 
+You can also grant access to another user to a former employee's OneDrive.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
You can also grant access to another user to access a former employee's OneDrive.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**